config.js

```diff
@@ -0,0 +1,9 @@
+exports.config = {
```

Comment:
this should go into `tests/cucumber/acceptance`. There can be other config files and there can be confusion